### DISCUSSION TITLE
How to "force rescan all library files" for external libraries via cli for docker?

### DISCUSSION THREAD

**Discussion by amgschnappi** [immich-app/immich#9058] [Q&A]
Hello,

I would like to "force rescan all library files" for external libraries via cli every week.
Currently I need to manually do it via gui.
How to do it via cli via docker?

Br

**Comment by bo0tzz** [comment]
There is no way to do this via the cli, but using the api should work: https://immich.app/docs/api/scan-library

**Reply by amgschnappi** [reply]
Too bad. :( I am not programmatically inclined and something similar to photoprism via cli would have been great. Will wait for this feature. What does Library watching in the settings do?
